README.md

```diff
@@ -134,7 +134,7 @@ $request_body = json_decode('{
 }');
 
 $apiKey = getenv('SENDGRID_API_KEY');
-$sg = new \SendGrid($apiKey);
+$sg = new \SendGrid\SendGrid($apiKey);
```

Comment:
I'm right there with you. I believe Client would conflict since the same namespace is being used in both this library and sendgrid/php-http-client.

I also don't like APIClient too much since it isn't a client, but a factory to build the client. I'm toying with that now to see what it looks like. I'll push the change to get your opinion.